broker/broker-undelivered-messages-ui/src/test/java/org/codice/ddf/broker/ui/UndeliveredMessagesTest.java

```diff
@@ -56,9 +56,10 @@
     private static final String RETRY_MESSAGE = "retryMessage";
 
     private final ObjectName objectName = new ObjectName(
-            "org.apache.activemq.artemis:type=Broker,brokerName=\"" + SystemBaseUrl.getHost()
-                    + "\",module=" + MODULE + ",serviceType=Queue,address=\"" + ADDRESS
-                    + "\",name=\"" + ADDRESS + "\"");
+            "org.apache.activemq.artemis:broker=\"" + SystemBaseUrl.getHost()
```

Comment:
That's correct. Artemis refactored their object name: https://issues.apache.org/jira/browse/ARTEMIS-953